addons/stock/models/stock_picking.py

```diff
@@ -1040,9 +1047,12 @@ def _pre_action_done_hook(self):
                 return pickings_to_immediate._action_generate_immediate_wizard(show_transfers=self._should_show_transfers())
 
         if not self.env.context.get('skip_backorder'):
-            pickings_to_backorder = self._check_backorder()
-            if pickings_to_backorder:
-                return pickings_to_backorder._action_generate_backorder_wizard(show_transfers=self._should_show_transfers())
+            pickings_to_backorder = self.filtered(lambda p: p.picking_type_id.create_backorder != 'never')._check_backorder()
+            # Do not generate a backorder confirmation wizard for the pickings
+            # that are configurated to generate backorders automatically (i.e. force_backorder is True in its picking type)
+            pickings_backorder_to_confirm = pickings_to_backorder.filtered(lambda picking: picking.picking_type_id.create_backorder == 'ask')
```

Comment:
Since we loop on each picking in `_check_backorder`, the condition could be into the loop itself and pass if its picking type config ins't `ask`
```python
        if picking.picking_type_id.create_backorder != 'ask':
            continue
```
That way we can avoid two `filtered` and keep the logic about that in `_check_backorder`